pkg/speccheck/builtin/process_service_instance_test.go

```diff
@@ -0,0 +1,368 @@
+package builtin
```

Comment:
These tests are mostly the same. I've modified some of them to ensure the annotation is always set (see previous comment).